### PR TITLE
Prepend money fields with "$"

### PR DESCRIPTION
**Pull request by jessieay**
* Move away from mask money js, which was causing a weird bug with
floats and saving values as zero.

screen shot 2017-08-25 at 1 51 05 pm

### CODE REVIEW

**Comment by jessieay** [comment]
[For a number of questions — Change free input to "Dollar Input" with "$" prepended and ".00" appended to all input.](https://trello.com/c/fmecPz0T/128-for-a-number-of-questions-change-free-input-to-dollar-input-with-prepended-and-00-appended-to-all-input)

**Comment by jessieay** [comment]
[Bug: money input not working for float fields](https://trello.com/c/dbpMAUh0/156-bug-money-input-not-working-for-float-fields)

File: app/helpers/mb_form_builder.rb [hunk 1]
```diff
@@ -34,25 +34,20 @@ def mb_input_field(
   def mb_money_field(
     method,
     label_text,
-    type: "text",
+    type: :tel,
```
Comment (adarsh) [inline app/helpers/mb_form_builder.rb]: Should this be `:tel`? Does that mean Telephone?

File: app/helpers/mb_form_builder.rb [hunk 2]
```diff
@@ -34,25 +34,20 @@ def mb_input_field(
...
     notes: [],
     options: {},
-    classes: [],
-    prefix: nil,
+    classes: ["form-width--med"],
+    prefix: "$",
```
Comment (adarsh) [inline app/helpers/mb_form_builder.rb]: Nice. This is super clear.

File: app/helpers/mb_form_builder.rb (hunk 1 above)
Comment (jessieay) [reply app/helpers/mb_form_builder.rb]: yeah it's so that on mobile the number keyboard is pulled up!

**Pull request merged**